autoload/lsc/reference.vim

```diff
@@ -143,6 +143,15 @@ function! s:showHover(result) abort
       let l:lines += split(item, "\n")
     endif
   endfor
+  if exists('g:lsc_preview_hover_popup') && has("patch-8.1.1513")
```

Comment:
Could this be something like `exists('*popup_atcursor')` instead of `has("patch-8.1.1513")`?

Reply:
ok
wait a moment fix.

Reply:
`exists('*popup_atcursor')`  instead `has("patch-8.1.1513")`